image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -22,9 +22,11 @@ image:
   scannerV4:
```

Comment:
Is there a reason why Scanner V4 is special here? I noticed that you didn't add a default `pullPolicy` to `main.image.scanner,scannerDb` in this file?